kpatch-build/lookup.c

```diff
@@ -414,19 +414,18 @@ bool lookup_local_symbol(struct lookup_table *table, char *name,
 			break;
 
 		if (sym->bind == STB_LOCAL && !strcmp(sym->name, name)) {
-			match = 1;
-			break;
+
+			if (result->objname)
+				ERROR("duplicate local symbol found for %s", name);
```

Comment:
Could we dump a bit more information here?
Like the address of both symbols? (And maybe the size)